### ISSUE TITLE
Prints errors to stdout instead of stderr

### ISSUE THREAD

**Issue by matthijskooijman** [popcornmix/omxplayer#780]
I just noticed that some errors are printed directly to stdout, rather than stderr.  In particular, I noticed this one: https://github.com/popcornmix/omxplayer/blob/f543a0d0e707ab56415f17b0ca6d397394ee8b63/omxplayer.cpp#L941

But searching for `printf` in the source turns up plenty of other errors.

It would be helpful to print errors to stderr instead, to better separate them from regular output.

**Comment by popcornmix** [comment]
I believe all messages from omxplayer are printed to stdout rather than stderr.

**Comment by matthijskooijman** [comment]
That's consistent with my observations (I just wrote "some" because I have not verified all messages, "some" <= "all" in my meaning :-p).

In any case, the question is unchanged: Wouldn't it be better to print errors to stderr?

**Comment by popcornmix** [comment]
Using stderr rather than stdout is mostly important when the app has a mode where stdout is used for piping data. omxplayer doesn't operate in that mode.

As omxplayer being deprecated and resources are being spent on improving vlc it's unlikely this will change. If it's important to you I'd consider a PR.

**Comment by matthijskooijman** [comment]
> Using stderr rather than stdout is mostly important when the app has a mode where stdout is used for piping data.

Fair point. I'm trying to show errors but not other output, but that's not essential.

> As omxplayer being deprecated 

Ah, missed that. Just started using it fairly recently for an unattended video player, which works rather well so far. Might look at VLC as an alternative at some point. Thanks!

> If it's important to you I'd consider a PR.

Well, it's not *that* important (enough things on my plate already), so I probably won't get around to that.

Feel free to close this is you want, or leave it open for others.
